packages/core/test/foundry/optimistic-assertor/OptimisticAsserter.Ownership.t.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
this is the key starting point that you guys should look at: this is how simple it should be to setup a test and run the base tests. I've re-structured things to pass around structs of contracts from each set of fixtures. see below.